src/sentry/static/sentry/app/views/settings/components/emptyMessage.tsx

```diff
@@ -6,15 +6,16 @@ import InlineSvg from 'app/components/inlineSvg';
 import TextBlock from 'app/views/settings/components/text/textBlock';
 import space from 'app/styles/space';
 
-const Wrapper = styled('div')`
+type WrapperProps = Pick<EmptyMessageProps, 'size'>;
+const Wrapper = styled('div')<WrapperProps>`
   display: flex;
   text-align: center;
   align-items: center;
   flex-direction: column;
   color: ${p => p.theme.gray4};
   padding: ${p => p.theme.grid * 4}px 15%;
   font-size: ${p =>
-    p.size && p.size.large ? p.theme.fontSizeExtraLarge : p.theme.fontSizeLarge};
```

Comment:
Does the previous invocation still work? (ie. `p.size.large`). I think using that interface would be preferable – that way the logic of what raw string represents "large" is encapsulated in one place, instead of all over the codebase.

Reply:
@mnoble I don't think the previous thing worked at all... (`p.size.large`). Just tested it out by changing the `size` prop on master, and it didn't do anything. The new version does work. It's because the `size` prop is a string and not an object

Reply:
@mnoble  Unless I'm missing something about how this `size` prop should work!

Reply:
> that way the logic of what raw string represents "large" is encapsulated in one place, instead of all over the codebase.

That's one of the advantage about typescript actually is that we have these string union types. This is actually preferable since using the typescript server with a IDE will suggest these string values :)

Reply:
`size` as a string seems correct, but I would do a search in the project for uses of `<EmptyMessage>` and look at how we use the `size` prop and confirm it should be a string.

Reply:
@nolachen Good catch! I think it is a bug. That being said, this will change the behavior when `size="large"` such as is `IntegrationAlertRules`.  Let's just make sure those views still look OK with this change.